axis.go

```diff
@@ -571,13 +631,13 @@ func log(x float64) float64 {
 	return math.Log(x)
 }
 
-// formatFloatTick returns a g-formated string representation of v
+//formatFloatTick returns a g-formated string representation of v
```

Comment:
please keep the space between `//` and `formatFloatTick`.

Reply:
Changed that. Sorry for removing the space. Was not done on purpose.